package.json

```diff
@@ -22,7 +22,9 @@
     "test": "vitest"
   },
   "dependencies": {
+    "@eth-optimism/contracts-ts": "^0.15.0",
     "@eth-optimism/core-utils": "^0.12.2",
+    "@roninjin10/rollup-chains": "^0.0.3",
```

Comment:
these should be in this repo isntead of a seperate pacakge

Reply:
roger

Reply:
Apparently not according to Jake they are better here to be more easily ported into viem